src/phoenix/db/bulk_inserter.py

```diff
@@ -60,13 +65,14 @@ def __init__(
             [] if initial_batch_of_evaluations is None else list(initial_batch_of_evaluations)
         )
         self._task: Optional[asyncio.Task[None]] = None
-        self._last_inserted_at: Optional[datetime] = None
+        self._last_inserted_at: LRUCache[ProjectRowId, datetime] = LRUCache(maxsize=100)
```

Comment:
might be more clear if the variable name is changed, like last_updated_at_by_project